src/main/scala/scalaz/parsers/alt/MonadErrorClass.scala

```diff
@@ -1,12 +1,16 @@
-package scalaz.parsers
+package scalaz.parsers.alt
```

Comment:
Why do you insist on `defs` and `alt` instead of just `tc` (as in scalaz) for all typeclasses? It isn't obvious what it means for `MonadError` to be in `alt` package...